project/Cargo.scala

```diff
@@ -0,0 +1,43 @@
+import sbt.Keys._
+import sbt._
+import sbt.internal.util.ManagedLogger
+
+import scala.sys.process._
+
+
+
+object Cargo {
```

Comment:
I think it would be helpful if the functions inside of this object got some documentation comments (even very short summaries) explaining their purpose.